zebra-consensus/Cargo.toml

```diff
@@ -5,21 +5,27 @@ authors = ["Zcash Foundation <zebra@zfnd.org>"]
 license = "MIT OR Apache-2.0"
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-zebra-chain = { path = "../zebra-chain" }
-zebra-state = { path = "../zebra-state" }
 chrono = "0.4.12"
+futures = "0.3.5"
+futures-core = "0.3.5"
 futures-util = "0.3.5"
-tower = "0.3.1"
+rand = "0.7"
+redjubjub = { git = "https://github.com/ZcashFoundation/redjubjub.git", branch = "main"}
+tokio = { version = "0.2", features = ["time", "sync", "stream"] }
+tower = "0.3"
+tower-batch = { path = "../tower-batch/" }
+tracing = "0.1.15"
+tracing-futures = "0.2.4"
+zebra-chain = { path = "../zebra-chain" }
+zebra-state = { path = "../zebra-state" }
```

Comment:
not sure it matters much but we might want to keep all the in-workspace deps separated from the others

Reply:
(this is marked-as-resolved-able, just wanted to mention it after seeing it)